src/Main.java

```diff
@@ -1,7 +1,16 @@
 public class Main {
 
     public static void main(String[] args) {
+        System.err.println("la potenza è" + pow(2, 4));
+    }
+
+    public static int pow(int base, int exponent) {
```

Comment:
Ci stavamo uniformando sul chiamare i metodi con get davanti. Quindi sarebbe da rinominare il metodo in getPow!